Broadcom/siteminder/tests/AuthAccept.json

```
@@ -0,0 +1,74 @@
+  },
+  "expected": {
+    "message": "AuthAccept HOST_XXX [23/Jul/2025:09:42:41 +0200] \"2.2.2.2 uid=toto,ou=Internal,ou=People,o=group\" \"domain.com GET /affwebservices/redirectjsp/redirect.jsp?RelayState=https%3A%2F%2Fproxy-auth.xxxxgroup.com%2Fmwg-internal%2Fde5fs23hu73ds%2Fplugin%3Ftarget%3DAuth%26reason%3DAuth%26ClientID%3D3664958249%26ttl%3D28800%26url%3DaHR0cHM6Ly9zcHJkYi5lc28taW8uY29tOjQ0Mw%2C%2C%26rnd%3D1753256545.1343803810.NIYDKGKubm1JCVNw4hvLJWZnoE9zPvw2tZ5DCvRz0LA%2C&SAMLRequest=fZLLcpswFIZfxaNFVzYIgQyoxh0CpraD09Zpk7E3HRUrhlZIVBc78dOXkOk0XbTL%2F1z%2FOd%2BZadryjqTW1GLLflqmzeix5UKTIZEAqwSRVDeaCNoyTUxFbtNNSZADSaekkZXkYJRqzZRppMik0LZl6papU1OxL9syAbUxnSau25c%2FPk1ov8oxNeVMH5W0nVPJ1m3Px0kjDFOCcvfA8INGfm1D%2F6DdjttjI94Zqo7MJM9G39C2e6sY1VL80RlvmDCrPPGn0yDGEQriIW4MT1AUQTgoq3hCl1tYLTfT8im%2BVMv1j12DeYUiQ%2B8ju0Ox%2BfD9E9ycx%2BOXNeKQeCH2EZ7iADueH%2FgR9CMPOjerXX79%2Ftp%2Ba711dndzDupTub7fC7mILx9PZ2T2OM9O2wss0zEYFVJVbLhyAh4o1wyMeq%2Fg6yJEMQwL7IfRIvPCqyhOiyDL0ysY5mGBir5Ma8tWQhsqTAIQRHgCwwnyP8OQBIggvAejO6Z0f%2Fw%2B7UAwnz2jI0OfegXz%2Fyzpb4Jg%2Fm9OM%2FfV6PmL%2Bvt55r8A&SMPORTALURL=https%3A%2F%2Fsso-idp.corp.xxxx%2Faffwebservices%2Fpublic%2Fsaml2sso\" [idletime=18000;maxtime=18000;authlevel=5;] [0]  [] []",
+    "event": {
+      "action": "AuthAccept",
+      "category": [
+        "authentication"
+      ],
+      "dataset": "smaccess",
+      "kind": "event",
+      "outcome": "success",
+      "reason": "Sm_Api_Reason_None",
+      "type": [
+        "allowed",
+        "start"
+      ]
+    },
+    "@timestamp": "2025-07-23T07:42:41Z",
+    "broadcom": {
+      "siteminder": {
+        "agent": {
+          "name": "domain.com"
+        },
+        "authlevel": "5",
+        "idle": "18000",
+        "maxtime": "18000",
+        "transaction": {
+          "id": "idletime=18000;maxtime=18000;authlevel=5;"
+        }
+      }
+    },
+    "client": {
+      "address": "2.2.2.2",
+      "ip": "2.2.2.2",
+      "user": {
+        "name": "uid=toto,ou=Internal,ou=People,o=group"
```

Can you split this value into `user.name` and `user.domain`? It will be easy to use for detection
```suggestion
        "name": "toto",
        "domain": "ou=Internal,ou=People,o=group"
```
see [Fortigate](https://github.com/SEKOIA-IO/intake-formats/blob/f77c3be37f3c132efdd46ef8d9c4d66e3a76ab75/Fortinet/fortigate/ingest/parser.yml#L400)